src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/DefaultAccessEgress.java

```diff
@@ -36,6 +36,11 @@ public int durationInSeconds() {
     return durationInSeconds;
   }
 
+  @Override
+  public boolean stopReachedOnBoard() {
```

Comment:
Ok - that is fine -  I will make a new method for this functionality.

Reply:
To be honest with you Hannes - I read the java docs and thought it suited this perfectly. I will revise the Javadoc to make it closer to its actual use.

Reply:
Yes, in the egress case it isn't described correctly. It should say that if it is an access, it is true if the access arrives onboard a vehicle to the stop, and in case of egress if the egress leaves the stop onboard a vehicle.